Makefile

```diff
@@ -24,92 +29,94 @@ DOCKER_BUILD ?= docker buildx build
 
 KIND_CLUSTER_NAME ?= containerd-wasm
 
+export
```

Comment:
typo?

Reply:
No, this means "export everything".
Since this is using nested `make` calls, I thought it would be simpler to export everything.